active/0000-cli-guides.md

```diff
@@ -0,0 +1,186 @@
+- A strike team channel will be created on a chat
+- A writing styleguide will be provided for contributors
+- Following a verson one release, writing work will be organized via normal GitHub issues.
+
+Since maintaining consistent voice and structure across a blank slate is a challenge, beta content for the core learning experience has already been drafted, including Basic Use guides and a tutorial for creating an addon from start to finish.
+
+The beta version of the CLI Guides content can be found at [ember-learn/cli-guides-source](https://github.com/ember-learn/cli-guides-source). The Markdown files there are rendered by [ember-learn/cli-guides-app](https://github.com/ember-learn/cli-guides-source). The app is currently [deployed on Netlify](https://sad-noyce-026d89.netlify.com/) for testing and UX validation, however it requires more work to resolve many known bugs.
+
+### User Personas
+
+The content layout should follow the progression of an Ember developer's
+learning experience. There are four main user personas for the 
+CLI documentation:
+
+1. A new or "typical" Ember CLI user - someone whose primary work is
+running common commands like `ember serve` and who has a "zero
+config" type of experience with Ember
+2. Power users - developers who make their own configurations to the
+build pipeline
+3. Beginner addon authors - those who are looking to build simple
+shared UI components, methods, or wrappers for existing npm libraries
+4. Advanced addon authors - those who dig into internals to make their
+addon work, or who are planning for broad extensibility
+
+### Table of Contents
+
+Applying these User Personas to the CLI content, the following topics layout emerges. "Beginner" topics will include links to later "Advanced"
+topics, similar to how the Guides link to the API docs.
+
+- Introduction 
+    - how to install ember cli 
+    - a very simple, short definition of what it is (the official way to create, build, and test an Ember app)
+    - Why is the CLI needed
+    - Guidance on learning path
+    - How to contribute
+- Basic use (explain options of each)
+    - CLI Commands: Explain how to use the `help` command and common commands like `ember new`, `ember server`, `ember generate`, etc. Each is explained briefly, together with an example usage and a link to the Main Ember Guides with more information about how to use those files.
+    - How to find and use addons
+    - How to use npm packages
+    - Installation and Upgrading the CLI (including a note about upgrading your app, with a link to more resources)
+    - feature flags & configurations
+- Advanced use
+    - shims
+    - broccoli
+    - custom blueprints
+    - CSS compilation
+    - Using another testing library
+    - more on dependencies
+    - more configurations
+- Writing Addons
```

Comment:
Would love to see two specific sections for addon authors:

- Configuration: There are two broad paths `environment.js` and `ember-cli-build.js` and then a lot of different directions addons that I have worked with seem to take for reading that configuration and acting on it.

- Nested Addons: there seem to be several approaches in the wild for dealing with finding the final application and configuration when addons are nested under other addons.

Reply:
Great idea, added!